backend/users/admin_views.py

```diff
@@ -0,0 +1,35 @@
+from asgiref.sync import async_to_sync
+from django.conf import settings
+from django.http import JsonResponse
+from pythonit_toolkit.service_client import ServiceClient
+
+SEARCH_USERS = """query SearchUsers($query: String!) {
+    searchUsers(query: $query) {
+        id
+        displayName
+    }
+}
+"""
+
+
+def users_autocomplete(request):
+    term = request.GET.get("term", "")
+    client = ServiceClient(
+        url=f"{settings.USERS_SERVICE}/internal-api",
+        service_name="users-backend",
+        caller="pycon-backend",
+        jwt_secret=settings.SERVICE_TO_SERVICE_SECRET,
+    )
+    client_execute = async_to_sync(client.execute)
```

Comment:
nit: we can add `client.execute_sync` to make this slightly nicer (for another PR anyway)